lib/Sema/MiscDiagnostics.cpp

```diff
@@ -1184,6 +1184,39 @@ bool swift::fixItOverrideDeclarationTypes(TypeChecker &TC,
     return true;
   };
 
+  // Check if overriding fails because we lack @escaping attribute on the function
+  // type repr.
+  auto checkTypeMissingEscaping = [&](Type overrideTy, Type baseTy,
+                                      SourceRange typeRange) -> bool {
+    // Fix-it needs position to apply.
+    if (typeRange.isInvalid())
+      return false;
+    auto overrideFnTy = overrideTy->getCanonicalType()->getAs<FunctionType>();
```

Comment:
I think the lesson from the other patch was that we should *not* attempt to match types or ext-info; if we have no better reason why the override failed then being two AnyFunctionTypes mismatching in `@escaping` is a good enough reason for the diagnostic.